src/sentry/models/organizationmembermapping.py

```diff
@@ -44,6 +47,9 @@ class OrganizationMemberMapping(Model):
     class Meta:
         app_label = "sentry"
         db_table = "sentry_organizationmembermapping"
-        unique_together = (("organization_id", "user"), ("organization_id", "email"))
+        unique_together = (
+            ("organization_id", "organizationmember_id", "user"),
+            ("organization_id", "organizationmember_id", "email"),
+        )
```

Comment:
```suggestion
        unique_together = (
            ("organization_id", "organizationmember_id"),
            ("organization_id", "user"),
            ("organization_id", "email"),
        )
```

Should we keep the old indexes and add a new one?